apps/dashboard/src/components/workflow-editor/steps/email/maily.tsx

```diff
@@ -1,30 +1,14 @@
 import { FormControl, FormField, FormMessage } from '@/components/primitives/form/form';
 import { useWorkflow } from '@/components/workflow-editor/workflow-provider';
-import { parseStepVariables } from '@/utils/parseStepVariablesToLiquidVariables';
 import { useFeatureFlag } from '@/hooks/use-feature-flag';
+import { parseStepVariables } from '@/utils/parseStepVariablesToLiquidVariables';
 import { cn } from '@/utils/ui';
 import { Editor } from '@maily-to/core';
-import {
```

Comment:
Nit-picky comment: With this refactoring, readers must open two files instead of one to understand the Maily props. I suggest we revert this change and keep everything in a single file.

Reply:
The reason being is, that I need to use maily configuration also in another component, and didn't wanted to copy paste the config json. Let me thing of maybe we can have a maily component separate from the form fields in which they are used.

Reply:
I agree, that would be better. Make Maily.tsx reusable and introduce steps/email/email-editor.tsx, following naming conventions of the other *-editor.tsx files.